### DISCUSSION TITLE
I need to set the keepAliveTimeout in Remix server but I can't find where or if it is possible. is it?

### DISCUSSION THREAD

**Discussion by lucioladen** [remix-run/remix#5753] [Q&A]
I'm having issues with an AWS load balancer. Remix server is closing the connection with the ELB as I understand default keepAliveTimeout for Node is 5 seconds. Any request taking more than 5 seconds is not working. Increasing aws load balancer idle timeout only is not enough, so I need to increase keepAliveTimeout of remix server. is it possible? can't find how to do it anywhere...

**Comment by chrisdostert** [comment]
Plus one. GCP https load balancers [require a timeout of greater than 600s](https://cloud.google.com/load-balancing/docs/https/#timeouts_and_retries) and I also can't find how/where to configure this in Remix server

**Reply by chrisdostert** [reply]
looks like this may be a dupe of https://github.com/remix-run/remix/discussions/5196 or at least highly related

**Comment by lili21** [comment]
I guess you can pick the express template. with the express server, you can config that.
